### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### Milestone 3
In this milestone:
**I have added the following:**

- Desktop Html
- Desktop CSS(style)
- Media query to toggle between screen sizes
- images and parse of the original Figma prototype

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,573 +1,1200 @@
 @import url(https://fonts.googleapis.com/css?family=Playfair+Display|Poppins);
```
Comment (carlos-ssh) [inline style.css:2]: - Kindly align items in small screen.
Screen Shot 2021-07-23 at 08 05 17

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @gabrielcoder247, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (bellom) [reply style.css:2]: @gabrielcoder247, we still need to work on the small screen alignment, and also, the hamburger seems not to appear on the small screen.

Screenshot from your app.

image

 
image



image




**Compare to the** [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9):
image

**Review by carlos-ssh — APPROVED** [review]
Hi @gabrielcoder247, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**